README.md

```diff
@@ -1,11 +1,16 @@
 # Google Cloud Datastore `ndb` Client Library
 
+[![stable](https://img.shields.io/badge/support-stable-gold.svg)](https://github.com/googleapis/google-cloud-python/blob/main/README.rst#stability-levels)
+[![pypi](https://img.shields.io/pypi/v/google-cloud-ndb.svg)](https://pypi.org/project/google-cloud-ndb/)
```

Comment:
The doc linked on the line above uses |PyPI-google-cloud-ndb| to get the version.  Can that be used here?

Reply:
I think that's just how RST does image links (you can see the image link and target link here are the same as the one here: https://source.corp.google.com/h/github/googlesource/codesearch/+/master:GoogleCloudPlatform/gcloud-python/README.rst;l=349?q=PyPI-google-cloud-ndb&ct=os